### PR TITLE
Linking "Unit of Work"

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

File: docs/en/tutorials/getting-started.rst [hunk 1]
```diff
@@ -642,7 +642,7 @@ Let's continue by creating a script to display the name of a product based on it
     echo sprintf("-%s\n", $product->getName());
 
 Next we'll update a product's name, given its id. This simple example will
-help demonstrate Doctrine's implementation of the UnitOfWork pattern. Doctrine
+help demonstrate Doctrine's implementation of the :ref:`UnitOfWork pattern<unit-of-work>`. Doctrine
```
Comment (phansys) [inline docs/en/tutorials/getting-started.rst]:
```suggestion
help demonstrate Doctrine's implementation of the :ref:`UnitOfWork pattern <unit-of-work>`. Doctrine
```

File: docs/en/tutorials/getting-started.rst [hunk 2]
```diff
@@ -1334,7 +1334,7 @@ call this script as follows:
     php create_bug.php 1 1 1
 
 See how simple it is to relate a Bug, Reporter, Engineer and Products?
-Also recall that thanks to the UnitOfWork pattern, Doctrine will detect
+Also recall that thanks to the :ref:`UnitOfWork pattern<unit-of-work>`, Doctrine will detect
```
Comment (phansys) [inline docs/en/tutorials/getting-started.rst]:
```suggestion
Also recall that thanks to the :ref:`UnitOfWork pattern <unit-of-work>`, Doctrine will detect
```

**Review by phansys — DISMISSED** [review]

File: docs/en/tutorials/getting-started.rst (hunk 1 above)
Comment (ThomasLandauer) [reply docs/en/tutorials/getting-started.rst]: Are you sure about the space?
Cause I already had some back and forth with the syntax at https://github.com/doctrine/orm/pull/9151, and now it finally works nice (see https://www.doctrine-project.org/projects/doctrine-orm/en/2.10/reference/events.html#events-overview)

Comment (phansys) [reply docs/en/tutorials/getting-started.rst]: Not really, just trying to mimic and keep consistence with the [existing links](https://github.com/doctrine/orm/search?l=reStructuredText&q=ref%3A).

Comment (ThomasLandauer) [reply docs/en/tutorials/getting-started.rst]: Yes, indeed, you're right: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref
But it obviously works without the space too ;-)

**Review by phansys — APPROVED** [review]

**Comment by ThomasLandauer** [comment]
Closing, as I now integrated it into https://github.com/doctrine/orm/pull/9154

**Pull request closed without merging**